libpod/container_internal_linux.go

```diff
@@ -2141,7 +2164,13 @@ func (c *Container) updateHosts(path string) error {
 	if err != nil {
 		return err
 	}
-	c.state.BindMounts["/etc/hosts"] = newHosts
+	
+	err = c.mountIntoRootDirs("/etc/hosts", newHosts)
```

Comment:
NIT: Combine these lines together.
if err := c.mountIntoRootDirs ...

Reply:
I wasn't completely sure if you meant I should remove the blank line here or turn this into a one-liner.
I decided to merely remove the blank line to adhere to the style of the rest of the function.